test/helpers/BuildMapper.test.tsx

```diff
@@ -1,54 +1,46 @@
-import { buildMapper } from "../../src/utils/helpers";
+import { buildDatetimeStatus } from "../../src/utils/helpers";
 import { BUILD } from "../testutils";
 
-const generateBuild = (status: string) => [
-  {
-    ...BUILD,
-    status
-  }
-];
-describe("buildMapper", () => {
+const generateBuild = (status: string) => ({
+  ...BUILD,
+  status
+});
+
+describe("buildDatetimeStatus", () => {
   it("should return an active build", () => {
-    const builds = generateBuild("COMPLETED");
-    const [mappedBuild] = buildMapper(builds, 1);
-    expect(mappedBuild.name).toContain("Active");
```

Comment:
`mappedBuild.name` should be equivalent to the value returned by `buildDatetimeStatus()`